### DISCUSSION TITLE
Too many tags on a snake break ladder formatting

### DISCUSSION THREAD

**Discussion by TheApX** [BattlesnakeOfficial/feedback#128] [Bug Reports]
If a snake has way too many tags, two bad things happen:

* Name is not visible
* Score is shifted outside of the table

I've just added a couple tags to my Nerdy Caterpillar, and this is how the 2021 Fall League [ladder](https://play.battlesnake.com/arena/fall-league-2021/) looks now:

![battlesnake-tags-bug](https://user-images.githubusercontent.com/30559107/138533281-a301de2a-8a29-4a27-9368-1240b3999da2.png)

-----

css is [awesome](https://css-tricks.com/wp-content/uploads/2021/04/css-is-awesome.jpg) ©

**Comment by naomi-lgbt** [comment]
To add on to the tag CSS:

![image](https://user-images.githubusercontent.com/63889819/138625158-7b5a4fda-ddd0-4153-99b2-1bdf1d5c071a.png)

They're invisible if highlighted.

**Reply by bvanvugt** [reply]
Great, thanks @nhcarrigan - we'll update this at the same time 👍

**Comment by bvanvugt** [comment]
Update - we've deployed changes today to address both issues: tag overflow, and default selection highlight under light text.

Thanks for the feedback!

**Reply by TheApX** [reply]
Thank you!

**Comment by coreyja** [comment]
Looks like the mobile view might still need some love in relation to this issue, just an FYI!

![image](https://user-images.githubusercontent.com/964771/140380028-0ad7794a-ee69-41d0-a705-41d64ac3aad3.png)

**Reply by bvanvugt** [reply]
Hmm yes. We'll see if we can get it fixed this week...
